bin/razor-admin

```diff
@@ -78,7 +86,7 @@ end
 
 # It is very important that razor/initialize isn't loaded until after
 # RACK_ENV has been set
-require_relative '../lib/razor/initialize'
+require 'razor/initialize'
```

Comment:
I can't find what I thought I read about it being deprecated, but there is some pretty solid guidance about when to use it and when not to use [it in this thread](https://bugs.ruby-lang.org/issues/8781).  Maybe I misremembered and conflated it with `autoload`. If I did, that's on me. Sorry for the confusion.